resources/patches/ungoogled-chromium/disable-google-host-detection.patch

```diff
@@ -7,18 +7,18 @@
  
  bool HasGoogleHost(const GURL& url) {
 -  static const char* kGoogleHostSuffixes[] = {
--      ".google.com",
```

Comment:
Looks like domain substitution tripped you up too huh

Reply:
Yeah, I fixed this patch manually in the last update and forgot to consider the domain substitution! Just uploaded a fix